packages/@uppy/core/src/index.js

```diff
@@ -1571,9 +1574,6 @@ class Uppy {
         const uploadID = this._createUpload(waitingFileIDs)
         return this._runUpload(uploadID)
       })
```

Comment:
Yes, because otherwise we were getting multiple error messages in the Informer, and the descriptive one was overridden by a less descriptive one later on. But I’ll add logging that second error to console 👌